README.md

```diff
@@ -236,6 +193,40 @@ while (keys.has_next()) {
 }
 ```
 
+### URLPattern
+
+Our implementation doesn't provide a regex engine and leaves the decision of choosing the right engine to the user.
+This is done as a security measure since the default std::regex engine is not safe and open to DDOS attacks.
```

Comment:
```suggestion
This is done as a security measure since the default std::regex engine is not necessarily safe and open to DDOS attacks.
```

(I don't think we should say that it is always unsafe. We don't know. It depends on the implementation.)